packages/synapse-react-client/src/style/components/entity_finder/_tree-view.scss

```diff
@@ -76,4 +76,8 @@
       font-size: large;
     }
   }
+
+  .optionsButton {
```

Comment:
You might be able to use the selector `.Header .MuiButton-root` so you don't have to worry about passing a `className` prop to the DropdownSelect